### ISSUE TITLE
The loss is nan

### ISSUE THREAD

**Issue by rabbicat30** [jaywonchung/BERT4Rec-VAE-Pytorch#22]
In some cases, such as the sequence length is short or the value of mask_prob is small, there will be a situation where the whole training sequence is not masked, and the loss at this time will be the value of nan, how to solve this situation? I don't want the loss to be a nan value, can I only adjust the value of prob?

**Comment by rabbicat30** [comment]
Because I want to do a comparison experiment, I want to make sure the value of mask_prob is the same
